### PR TITLE
fix(CLDX-190): changes as per testing sign-binaries task with pipeline

### PR DESCRIPTION
**Pull request by pkhander**
This commit fixes tasks releated to push-to-dev-portal pipeline

### CODE REVIEW

File: tasks/extract-binaries-from-image/extract-binaries-from-image.yaml
Comment (johnbieren) [inline tasks/extract-binaries-from-image/extract-binaries-from-image.yaml:1]: forgot to bump version string

File: tasks/prepare-exodus-params/README.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+exodus-gw is a microservice dedicated to writing data onto the CDN.
 
 ## Parameters
 
 | Name     | Description                                                                                                                 | Optional | Default value |
 |----------|-----------------------------------------------------------------------------------------------------------------------------|----------|---------------|
 | dataPath | Path to the merged data JSON file generated by collect-data task and containing the exodus configuration options to use     | No       | -             |
 
+## Changes in 0.2.2
+* Use `cdn.env` instead of `exodus.env` to align RPA data with other pipelines
```
Comment (johnbieren) [inline tasks/prepare-exodus-params/README.md:15]: Whatever *is* read from the exodus key should be in https://github.com/konflux-ci/release-service-catalog/blob/development/schema/dataKeys.json

File: tasks/publish-to-cgw/README.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ Tekton task to publish content to Red Hat's Developer portal using pubtools-cont
 | cgwHostname | The hostname of the content-gateway to publish the metadata to  | yes      | https://developers.redhat.com/content-gateway/rest/admin |
 | cgwSecret   | The kubernetes secret to use to authenticate to content-gateway | yes      | publish-to-cgw-secret |
 
+## Changes in 0.2.3
```
Comment (johnbieren) [inline tasks/publish-to-cgw/README.md]: `s/3/4/`

**Review by johnbieren — CHANGES_REQUESTED** [review]
Your unit tests aren't passing

File: tasks/prepare-exodus-params/README.md (hunk 1 above)
Comment (pkhander) [reply tasks/prepare-exodus-params/README.md:15]: cdn is already part of the schema here! so this change also aligns us with that! 

https://github.com/konflux-ci/release-service-catalog/blob/development/schema/dataKeys.json#L449-L456

Comment (johnbieren) [reply tasks/prepare-exodus-params/README.md:15]: Yeah, I am saying if the exodus key, not cdn key, is still relevant for anything, it should be in the schema (if this change removes exodus entirely, nothing to do)

**Review by johnbieren — APPROVED** [review]
/ok-to-test

**Comment by johnbieren** [comment]
This PR has nothing to do with e2e

**Pull request merged**